doc/contribute_models.md

```diff
@@ -0,0 +1,71 @@
+# How to Contribute SQLFLow Models
+
+This guide will introduce how to contribute to SQLFlow models. If you haven't checked [Define SQLFLow Models](/doc/customized+model.md), please check it out first.
+
+## Add an SQLFlow Model
+
+1. Open the [SQLFlow models repo](https://github.com/sql-machine-learning/models) on your web browser, and fork the official repo to your account.
+
+1. Clone the forked repo on your hosts:
+
+    ``` bash
+    > git clone https://github.com/<Your Github ID>/models.git
+    ```
+
+1. You can add a new mode definition Python script under the folder [sqlflow_models](/sqlflow_models). For example, adding a new Python script `mydnnclassfier.py`:
+
+    ``` text
+    `-sqlflow_models
+        |- dnnclassifier.py
+        `- mydnnclassifier.py
+    ```
+
+1. Define your model named `MyDNNClassifier`:
+
+    ``` python
+    import tensorflow as tf
+
+    class MyDNNClassifier(tf.keras.Model):
+    def __init__(self, feature_columns, hidden_units=[10,10], n_classes=2):
+        ...
+    ```
+
+1. Import `MyDNNClassfier` in [sqlflow_models/__ini__.py]:
+
+    ``` python
+    ...
+    from .mydnnclassfier import MyDNNClassifier
+    ```
+
+## Integration test with SQLFlow gRPC server
+
+If you have developed a new model, please perform the integration test with the SQLFlow gRPC server to make sure it works well with SQLFlow.
+
+1. Launch an SQLFlow all-in-one Docker container
+
+    ``` bash
+    cd ./models
+    > docker run --rm -it -v $PWD:/models -p 8888:8888 sqlflow/sqlflow
+    ```
+
+1. Update `sqlflow_models` in the SQLFlow all-in-one Docker container:
+
+    ``` bash
+    > docker exec -it <container-id> pip install -U /models
```

Comment:
`pip install -U` only checks the version, need to uninstall and install it in the container.

Reply:
`pip install -U /models` works well, but throw an exception for the first execution:
``` text
    return self._has(path)
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1845, in _has
    return zip_path in self.zipinfo or zip_path in self._index()
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1722, in zipinfo
    return self._zip_manifests.load(self.loader.archive)
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1679, in load
    mtime = os.stat(path).st_mtime
FileNotFoundError: [Errno 2] No such file or directory: '/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/sqlflow_models-0.0.1-py3.6.egg'```

I'm not sure what is the best practices, will test it later.